bin/run_mtaircompressor.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# This file is part of AirCompressors.
```

Comment:
Please change `AirCompressors` to the project name `ts_MTAirCompressor` here and in all other files.

Reply:
Changed (hopefully everywhere). Was caused by originally naming it AirCompressors